Resources/Prototypes/_Goobstation/DeviceLink/source_ports.yml

```
@@ -0,0 +1,5 @@
+- type: sourcePort
+  id: MaterialSilo
+  name: signal-port-name-material-silo
+  description: signal-port-description-material-silo
+  defaultLinks: [ MaterialSiloUtilizer ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)